bootstrap/bootstrap.go

```diff
@@ -59,12 +58,43 @@ const (
 	apiTypeSignin  = "signin"
 )
 
-// bootstrap is a data structure to hold configuration required to start
+// Defaults.
+const (
+	DefaultIdentifierClientPath = "./identifier-webapp"
+	DefaultSigningKeyID         = "default"
+	DefaultSigningKeyBits       = 2048
+)
+
+//Stringly typed application config, represents the user accessible config params
+type Config struct {
+	ISS                            string
+	IdentityManager                string
+	URIBasePath                    string
+	SignInUri                      string
+	SignedOutUri                   string
+	AuthorizationEndpointURI       string
+	EndsessionEndpointURI          string
+	Insecure                       bool
+	TrustedProxy                   []string
+	AllowScope                     []string
+	AllowClientGuests              bool
+	AllowDynamicClientRegistration bool
+	EncryptionSecretFile           string
+	Listen                         string
+	IdentifierClientPath           string
+	IdentifierRegistrationConf     string
+	IdentifierScopesConf           string
+	SigningKid                     string
+	SigningMethod                  string
+	SigningPrivateKeyFiles         []string
+	ValidationKeysPath             string
+	CookieBackendUri               string
```

Comment:
Consistency - please change `Uri` to `URI`. I think the linter should also complain here.